Sources/Pokepay/BankAPI/Terminal/UpdateTerminal.swift

```diff
@@ -1,14 +1,14 @@
 import APIKit
 
 extension BankAPI.Terminal {
-    struct Update: BankRequest {
+    public struct Update: BankRequest {
```

Comment:
could this work if we make the extension itself public?
because the other part of the codes is implemented that way.

Reply:
Because we'll use it in [here](https://github.com/pokepay/flutter-sdk-internal/blob/9cde3351e7656b527461e7c23c48fbab83773b8e/ios/Classes/SwiftPokepaySdkPlugin.swift#L489)